### ISSUE TITLE
Config file parser chokes on comments

### ISSUE THREAD

**Issue by thehunmonkgroup** [yassen-itlabs/py-linux-traffic-control#9]
See below:

```
root@webrtc-test-0.stirlab.net$/var/local/git: cat /etc/pyltc.profiles 
[4g-sym]
clear
iface eth0
dclass tcp:443:512kbit
dclass tcp:8080:512kbit
dclass tcp:8082:512kbit
dclass udp:10000-35000:512kbit

; vi: ft=dosini

root@webrtc-test-0.stirlab.net$/var/local/git: pyltc profile 4g-sym -v
Args: (config=None, ingress=None, profile_name='4g-sym', subparser='profile', verbose=True, version=False)
Using config file '/etc/pyltc.profiles'
Profile args: ['tc', '--clear', '--iface', 'eth0', '--dclass', 'tcp:443:512kbit', '--dclass', 'tcp:8080:512kbit', '--dclass', 'tcp:8082:512kbit', '--dclass', 'udp:10000-35000:512kbit', '--;', 'vi:', 'ft=dosini']
usage: pyltc [-h] [-V] {profile,tc} ...
pyltc: error: unrecognized arguments: --; vi: ft=dosini
```

Also tried a comment with a leading #, which resulted in the same error.

**Comment by yassen-itlabs** [comment]
Yep, this is a known issue. As this is NOT a standard .ini format, we may decide what to support - semicolon or a hash sign, or maybe both?

**Comment by thehunmonkgroup** [comment]
I'd say support both, if it's not too hard.

**Comment by thehunmonkgroup** [comment]
curious what you're using to parse the current config file format? i would think python has some built in parsing library. i'm really not attached to any specific config file format, just something simple that works.

**Comment by yassen-itlabs** [comment]
There's the standard libary .ini ConfigParser but it does not support repeating keys (like dclass in our case) so I wrote a custom one. It's not hard to get that working (the comments), will do that at first convenience.

**Comment by yassen-itlabs** [comment]
Resolved with 45d9c8743d561858bc6d3f4680bb1ace2895126b, closing.
